sdks/nodejs-client/index.js

```diff
@@ -89,10 +154,21 @@ export class DifyClient {
         params,
         headers,
         responseType: "json",
+        validateStatus: false,
       });
     }
 
-    return response;
+    if (!response) {
+      throw new Error("No response from server");
+    }
+
+    if (response?.data.code !== undefined && response.data.code !== 200) {
+      throw new Error(
+        `${response.data.status} ${response.data.code}: ${response.data.message}`
+      );
+    }
+
+    return response.data;
```

Comment:
As a API library, I think returning data instead of the full response is more friendly for developers. But this is a breaking change, and can impact other clients. What do you think?

Reply:
I think it's okay to update it.